CheckForVitals.cpp

```diff
@@ -0,0 +1,66 @@
+#include "./CheckForVitals.h"
+#include "./PrintMessage.h"
+
+const float BASE_TOLERANCE = 0.015f;
+
+
+
+bool isBelowLowerLimit(float value, float lowerLimit, const std::string& message) {
+    if (value < lowerLimit) {
+        displayLoaderAndMessage(message);
+        return true;
+    }
+    return false;
+}
+
+bool isApproachingLimit(float value, float limit, float tolerance, const std::string& message) {
+    if (value < limit + tolerance) {
+        displayLoaderAndMessage(message);
+        return true;
+    }
+    return false;
+}
+
+bool isAboveUpperLimit(float value, float upperLimit, const std::string& message) {
+    if (value > upperLimit) {
+        displayLoaderAndMessage(message);
+        return true;
+    }
+    return false;
+}
+
+bool isExcedingLimit(float value, float upperLimit, float tolerance,
+    const std::string& message) {
+    if (value > upperLimit - tolerance) {
+        displayLoaderAndMessage(message);
+        return true;
+    }
+    return false;
+}
+
+bool checkAndDisplay(float value, float lowerLimit, float upperLimit,

```

Comment:
This is a boolean function. Recommend to name it with `is` or `are`, like `vitalsAreInRange`
However, that would miss the display part.
Such difficulty in naming would usually give an opportunity for refactoring - to achieve Single Responsibility.
E.g., in this case- changes in the logic (like `<` becoming `<=`) and addition of a display method - both would affect the same set of functions.
Try doing it in stages - like first classify with logic, then translate the classification into a message. Each of those functions will be easier to name.